docs/recipes/ember.md

```diff
@@ -0,0 +1,147 @@
+# Usage with Ember
+
+Ember follows a similar pattern to [Vue](./vue.md):
+
+- The machine can be defined externally;
+- The service is placed as a property of the component;
+- State changes are observed via `service.onTransition(state => ...)`, where you set some data property to the next `state`;
```

Comment:
I don't think I'd use the word service here, as that already means something very specific in the ember ecosystem :/

Reply:
How about `interpreted` or `runningMachine` or something?

Reply:
Call it an `actor` even if you want!

Reply:
In my projects I called it `this.machine` knowing that it is actually an interpreted machine. I also push the actual machine to a `createMachine` factory function.

I would want these docs to be as accessible as possible so I would vote for the most informative one. Do we have a vote?

Reply:
I went with `interpreter` and `toggleInterpreter`